core/txpool/subpool.go

```diff
@@ -78,8 +78,9 @@ type PendingFilter struct {
 	BlobFee     *uint256.Int // Minimum 4844 blobfee needed to include a blob transaction
 	GasLimitCap uint64       // Maximum gas can be used for a single transaction execution (0 means no limit)
 
-	OnlyPlainTxs bool // Return only plain EVM transactions (peer-join announces, block space filling)
-	OnlyBlobTxs  bool // Return only blob transactions (block blob-space filling)
+	OnlyPlainTxs  bool // Return only plain EVM transactions (peer-join announces, block space filling)
+	OnlyBlobV0Txs bool // Return only V0 encoded blob transactions (block blob-space filling)
+	OnlyBlobV1Txs bool // Return only V1 encoded blob transactions (block blob-space filling)
```

Comment:
Changing this API seems overkill since it is only useful for a short period around the fork boundary. I would just skip over `v0` blob sidecars in the miner worker after Osaka and we can remove it again later on after the fork.

Reply:
worker does not interface directly to the blobpool, only to the txpool, so I think getting a sidecar version would require even more API changes. It would either need to relay a call to the blobpool (PendingV1Only), or we would need to add blob version info to the LazyTransaction. Both seems messy to me compared to this.